### DISCUSSION TITLE
Simpler Install

### DISCUSSION THREAD

**Discussion by LDprg** [winapps-org/winapps#8] [Ideas]
Maybe we should consider following things:

- Installer for host
- Installer for vm
- preconfigured vm (if possible)
- more automation

The install of the legacy winapps was quite complicated and not automated, we should consider improving this.

**Comment by jjgalvez** [comment]
For hosts if it's a central install we should create a proper package, for local install an installer would be great. Best of both worlds create packages that install the installer to manage local instances.

**Comment by oskardotglobal** [comment]
This is also one of the things I wanted to do with the GUI, where there's some kind of setup wizard to create the VM. But for the CLI you should definetly be able to pass like CPU, RAM & storage limits and have the rest happen automatically.

**Reply by fbartels** [reply]
For my own use case it would be important that the installer can also simply connect to an already existing vm.

Some form of automated install of windows and applications on it would however be nice.

**Comment by oskardotglobal** [comment]
https://github.com/quickemu-project/quickemu
This might be useful

**Reply by fbartels** [reply]
Yes, building upon it will simplify the installation for users (don't have to take care of qemu). https://github.com/winapps-org/winapps/discussions/9 Is relevant in this context as well.
